src/url-replacements.js

```diff
@@ -94,6 +97,38 @@ class UrlReplacements {
             Promise.resolve());
       });
     });
+
+    this.set_('TIMESTAMP', () => {
+      return new Date().getTime();
+    });
+
+    this.set_('TIMEZONE', () => {
+      return new Date().getTimezoneOffset();
+    });
+
+    this.set_('SCROLL_TOP', () => {
+      return viewportFor(this.win_).getScrollTop();
```

Comment:
I think this is good, but we will need to put all SCROLL ones in vsync/promise. We will need to add the following method to vsync class:
```
/**
 * @param {function():*} measurer
 * @return {!Promise<*>}
 */
measurePromise(measurer) {
  return new Promise(resolve => {
    this.measure(() => {
      resolve(measurer());
    });
  });
}
```

Then we can simply call everywhere here:

```
  this.set_('SCROLL_LEFT', () => {
    return vsyncFor(this.win_).measurePromise(
          () => viewportFor(this.win_).getScrollLeft());
  })
```

Malte's latest PR already allows promise use here.